pkg/probe/fortiswitch_port_stats.go

```diff
@@ -0,0 +1,198 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func probeSwitchPortStats(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		mSwitchStatus = prometheus.NewDesc(
+			"fortiswitch_status",
+			"Whether the switch is connected or not",
+			[]string{"vdom", "name", "fgt_peer_intf_name", "connection_from", "state"}, nil,
```

Comment:
This meric is not like the rest. The fgt_peer_intf_name and connection_from is not clear to me what this metric is. Care to add it to the description or as a comment in code?